testsuite/functional-testsuite/src/test/script/prepare_siestajs.groovy

```diff
@@ -50,5 +51,12 @@ ant.unzip(src: artifacts['org.sonatype.nexus:nexus-siestajs-testsupport'].file,
     include(name: 'ft-overlay/**')
   }
 }
+
+// Extract common test file resources
+ant.unzip(src: artifacts['org.sonatype.nexus.testsuite:nexus-modern-testsuite'].file, dest: itDir) {
```

Comment:
If we enhanced `TestDataRule` from goodies-testsupport to work with multiple folders that would avoid the need to unpack the shared test resources under 'src/test/it-resources'. Alternatively we could change the `TestDataRule` declaration at https://github.com/sonatype/nexus-oss/blob/master/testsupport/nexus-pax-exam/src/main/java/org/sonatype/nexus/pax/exam/NexusPaxExamSupport.java#L90 to use a folder under the build directory and then copy over any local or shared test resources.